Makefile

```diff
@@ -1,7 +1,7 @@
 PIPELINE_FAMILY := general
 PIPELINE_PACKAGE := general
 PACKAGE_NAME := prepline_${PIPELINE_PACKAGE}
-PIP_VERSION := 22.1.2
+PIP_VERSION := 22.2.1
```

Comment:
nit: i've been thinking to avoid pip bumps until we bump pip everywhere

Reply:
ahh, I was bumping for consistency with [the unstructured repo](https://github.com/Unstructured-IO/unstructured/blob/ef9fb79ed482a5d2f8694fbbbcac3b81bbb1b85d/Makefile#L2).

but I am seeing that [other pipelines](https://github.com/Unstructured-IO/pipeline-sec-filings/blob/main/Makefile#L4) are still on 22.1.2. will revert